CHANGELOG.md

```diff
@@ -64,3 +64,4 @@
 [fixed]: http://nrel.github.io/OpenStudio-user-documentation/img/fixed.svg "Fixed"
 [improved]: http://nrel.github.io/OpenStudio-user-documentation/img/improved.svg "Improved"
 [removed]: http://nrel.github.io/OpenStudio-user-documentation/img/removed.svg "Removed"
+[changed]: ExteriorLights::schedule now returns boost::optional<Schedule> instead of Schedule
```

Comment:
Under the changes in 1.6.3 (actually now I think your changes should be listed under a new 1.7.0 release).  Newest stuff is at the top of the changelog